.azure/pipelines/quarantined-pr.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     - script: ./eng/build.cmd -ci -nobl -noBuildRepoTasks -noRestore -test -all -noBuildJava -noBuildNative
               -projects eng\helix\helix.proj /p:RunQuarantinedTests=true /p:IsRequiredCheck=true /p:IsHelixJob=true
               /p:BuildInteropProjects=true /p:RunTemplateTests=true
-              /p:CrossgenOutput=false /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
+              /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
```

Comment:
This isn't a particularly interesting change given we _expect_ failures in the quarantined runs. Suggest instead removing `/p:CrossgenOutput=false` from the 'Tests: Helix x64' job's build steps in ci.yml. If the `crossgen2`'d assemblies are platform-specific with the command arguments you're currently using, I'd expect to see a lot of failures on Linux and macOS work items (ProjectTemplates.Test in particular) submitted from that job without the property setting.